packages/core/test/temporal/TemporalTest.js

```diff
@@ -113,4 +113,10 @@ describe('js-joda Temporal', () => {
             expect(new BasicYearMock(2).equals(result)).to.equal(true);
         });
     });
+
+    describe('valueOf', () => {
+        it('should throw an exception', () => {
+            expect(()=> new BasicYearMock(1).valueOf()).to.throw(TypeError);
+        });
+    });
```

Comment:
Does it make sense to add a coercing test here as well ?

Reply:
My reasoning for this is that `BasicYearMock` is a minimal implementation of `Temporal`, so this tests that the behavior is really inherited for every `Temporal` and not only to the "standard" ones (like `LocalDate`, `Instant`, etc).